kmod/src/triggers.c

```diff
@@ -93,13 +93,9 @@ int scoutfs_setup_triggers(struct super_block *sb)
 		goto out;
 	}
 
-	for (i = 0; i < ARRAY_SIZE(triggers->atomics); i++) {
-		if (!debugfs_create_atomic_t(names[i], 0644, triggers->dir,
-					     &triggers->atomics[i])) {
-			ret = -ENOMEM;
-			goto out;
-		}
-	}
+	for (i = 0; i < ARRAY_SIZE(triggers->atomics); i++)
+		debugfs_create_atomic_t(names[i], 0644, triggers->dir,
```

Comment:
The commit message says that we still want to check the result on older kernels.. but it's not?  That's confusing and makes me worry that something was forgotten, etc.  If we want to check, I guess we add compat wrappers that return 0 on newer.  But I'm fine with not checking, honestly.  Just make the patch make sense either way.

Reply:
Easy solution found: in the upstream commit, greg tells us exactly what to do (stop checking). I'm just going to do that (avoiding the need for kc).